spark/src/test/java/io/spark/ddf/etl/TransformationHandlerTest.java

```diff
@@ -88,7 +88,7 @@ public void testReservedFactor() throws DDFException {
 
     Assert.assertEquals(31, ddf.getNumRows());
     Assert.assertEquals(9, ddf.getNumColumns());
-    Assert.assertEquals("test123", ddf.getColumnName(8));
+    Assert.assertEquals("test123", ddf.getColumnName(0));
     Assert.assertEquals(9, ddf.VIEWS.head(1).get(0).split("\\t").length);
```

Comment:
I see that new columns in transform using Hive are added to the front, while the new ones in transform using R are added to the back. We should be consistent. To me, adding them to the back is more preferred.